### PR TITLE
Found Inventory Objects Overlay

### PR DESCRIPTION
**Pull request by xesf**
**Preview here:** https://pr-362.lba2remake.net

### CODE REVIEW

**Review by slwilliams — APPROVED** [review]
lgtm, although I didn't test because looks like we didn't get a preview link.

File: src/model/inventory.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        files,
+        bodies: [],
+        texture: loadTextureRGBA(files.ress.getEntry(6), palette),
+        mesh: null,
+        boundingBox: null,
+        boundingBoxDebugMesh: null,
+        materials: []
+    };
+
+    const body = loadBody(model, model.bodies, invIdx, null);
```
Comment (agrande) [inline src/model/inventory.ts:70]: Oh, so you can actually call loadBody() directly, that's nice :)

**Review by agrande — APPROVED** [review]
That's a pretty big commit! Nicely done!
I like the generic overlay stuff.

**Comment by agrande** [comment]
> lgtm, although I didn't test because looks like we didn't get a preview link.

Yeah, seems like I broken that when trying to fix the autopush stuff, ugh, my server code is too brittle.
I reloaded the server and the PR preview is now available (not for long though, I'll check it and merge if it works).

**Pull request merged**